src/store/entities.js

```diff
@@ -56,7 +56,12 @@ export const defaultColumns = () => [
     title: 'Group',
     props: { width: 10 },
     // eslint-disable-next-line camelcase
-    renderFunc: (groups) => (isEmpty(groups) ? 'N/A' : groups[0].name), // currently, one group at maximum is supported
+    renderFunc: (groups) =>
+      isEmpty(groups) ? (
+        <div className="pf-v5-u-disabled-color-100">No group </div>
```

Comment:
It looks like in the ticket they explicitly asked for the 200 variant. It also gives more contrast compared to normal black text

```suggestion
        <div className="pf-v5-u-disabled-color-200">No group</div>
```